src/e2e/app.ts

```diff
@@ -107,6 +107,17 @@ export class App {
       await dialog.accept();
     });
 
+    const logsPath =
+      "reports/screenshots/" + expect.getState().currentTestName + ".txt";
+    page.on("console", (msg) => {
+      fs.appendFile(logsPath, msg.text(), (err) => {
+        if (err) {
+          // Log file error.
+          console.log("Log file error: ", err.message);
```

Comment:
I think this is fine to be best effort with a logged error, but let's at least do console.error.

Reply:
[`109f052`](https://github.com/microbit-foundation/python-editor-next/pull/474/commits/109f052b09bee1277305477bdc62a428562bf793)